example/lib/src/file_picker_demo.dart

```diff
@@ -33,11 +55,32 @@ class _FilePickerDemoState extends State<FilePickerDemo> {
         .addListener(() => _extension = _fileExtensionController.text);
   }
 
+  Widget _buildFilePickerResultsWidget(
+      {required int itemCount,
+      required Widget Function(BuildContext, int) itemBuilder}) {
+    final ScrollController scrollController = ScrollController();
```

Comment:
This scroll controller is not disposed. I would just omit the scroll controller / scrollbar? It's only a sample anyway.